omnibus/config/templates/datadog-agent-mac-app/Info.plist.erb

```diff
@@ -32,5 +32,7 @@
     <string>Copyright (c) 2009-<%= year %>, Datadog Inc.</string>
     <key>NSPrincipalClass</key>
     <string>NSApplication</string>
+    <key>NSLocalNetworkUsageDescription</key>
+    <string>Datadog need access to local network services to function"</string>
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏

```suggestion
    <string>Datadog needs access to local network services to function"</string>
```